From 5c9c3618a570d6649aaf707265627c76cdb7b031 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 12 Mar 2018 22:51:26 +0100
Subject: [PATCH 1/1] MAINTAINERS: ARM FREESCALE IMX: remove invalid path

arch/arm/cpu/armv7/mx*/ does not relate to any existing directory.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 MAINTAINERS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index d4900e000d2..ba6ff9a0859 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -111,7 +111,6 @@ S:	Maintained
 T:	git git://git.denx.de/u-boot-imx.git
 F:	arch/arm/cpu/arm1136/mx*/
 F:	arch/arm/cpu/arm926ejs/mx*/
-F:	arch/arm/cpu/armv7/mx*/
 F:	arch/arm/cpu/armv7/vf610/
 F:	arch/arm/mach-imx/
 F:	arch/arm/include/asm/arch-imx/
-- 
2.16.1

